tests/unit/Cryptography.Tests.ps1

```diff
@@ -52,4 +52,20 @@ Describe 'New-PodeSalt' {
         Mock Get-PodeRandomBytes { return @(10, 10, 10) }
         New-PodeSalt -Length 3 | Should Be 'CgoK'
     }
+}
+
+Describe 'Test-PodeJwt' {
```

Comment:
Along with moving the function to Authentication.ps1, this'll need relocating as well

Reply:
done